src/renderers/shared/reconciler/ReactChildReconciler.js

```diff
@@ -25,6 +27,25 @@ var shouldUpdateReactComponent = require('shouldUpdateReactComponent');
  */
 var ReactChildReconciler = {
 
+
+  _instantiateChild: function(childInstances, child, name) {
```

Comment:
Let's move this to a module level function so that it is not exported. Ensures that it can't be overridden, minimizes better etc.